src/main.js

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { render as reactDomRender } from 'react-dom';
+import superagent from 'superagent';
+import './style/main.scss';
+
+const apiUrl = 'http://www.reddit.com/r/';
+
+class RedditSearchForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchFormBoard: '',
+      searchFormLimit: '',
+    };
+
+    this.handleRedditBoardChange = this.handleRedditBoardChange.bind(this);
+    // this.handleSubmit = this.handleSubmit.bind(this);
+    // this.handleRedditLimitChange = this.handleRedditLimitChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleRedditBoardChange(event) {
+    this.setState({
+      searchFormBoard: event.target.value,
+    });
+  }
+
+  // handleRedditLimitChange(event) {
+  //   console.log(event.target.value);
+  //   this.setState({
+  //     renderTopicsList: event.target.value,
```

Comment:
`this.state` does not currently have a property named `renderTopicsList` in your `RedditSearchForm` component, consider changing this to `searchFromLimit`.